documentation/source/getting_started/04_learning.rst

```diff
@@ -0,0 +1,92 @@
+========
+Learning
+========
+
+*This is a brief overview of a learning task. Requirements for a more specific task may vary substantially based on the goals or the data available to you.*
+
+We now have ``bk.txt`` and ``facts.txt`` as a result of the previous step. In order to get show some results, we will construct a toy data set from predicates easily available in the facts file.
+
+The *Delcaration of Independence* contains a set of phrases called the "List of Grievances", where the Founders spell out the 27 violations by King George III.
```

Comment:
Typo Declaration

Reply:
Great catch: "De**lc**aration" =/= "De**cl**aration". Same deal, I'll resolve on Friday.

Reply:
Resolved in e5a6607e8558dd695f076778231d0ab6b34cfbfe